go/arrow/bitutil/Makefile

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this converts rotate instructions from "ro[lr] <reg>" -> "ro[lr] <reg>, 1" for yasm compatibility
```

Comment:
From what I've seen, the compiling has to be done on linux/macOS because it needs clang and yasm, but the resulting plan9 assembly from using `c2goasm` will be portable. Go's toolchain uses plan9 assembly as an intermediate before generating the platform specific instructions. As a result, windows will use the assembly version also, not the pure Go version (unless the `noasm` tag is set when compiling)